src/schema/submission.yaml

```diff
@@ -425,40 +434,42 @@ classes:
       - isolation
       - storage
       - xref
-      - condition
       - biospecimen
+      - condition
     slot_usage:
       name:
         description: >-
-          Name of the sample (eg:GHGAS_Blood_Sample1 or GHGAS_PBMC_RNAseq_S1).
+          A descriptive name of the Sample (e.g. GHGAS_Blood_Sample1 or GHGAS_PBMC_RNAseq_S1).
+          Please do not enter any personally identifiable data!
```

Comment:
Shall we change it to "This field **must not** include any personally identifiable data."? For the spreadsheet, I understand that directly addressing the user is useful. But on the documentation page, the phrase might not be accurate.